desktop/src/main/java/bisq/desktop/main/support/dispute/DisputeView.java

```diff
@@ -139,7 +142,7 @@
     public DisputeView(DisputeManager<? extends DisputeList<? extends DisputeList>> disputeManager,
                        KeyRing keyRing,
                        TradeManager tradeManager,
-                       CoinFormatter formatter,
+                       @Named(FormattingUtils.BTC_FORMATTER_KEY) CoinFormatter formatter,
```

Comment:
I think in this case it's unnecessary, because this constructor is not annotated with `@Inject`

Reply:
That is what I thought too, which is why I left these out originally. But it turns out `@inject` is not infact mandatory and the program crashes on these views without this annotation. Thanks to @stejbac for discovering this.

Reply:
Are you sure that app crash when you ommit `@Named` annotation on this particular file (`DisputeView`) ? It's an abstract class and it does not "participate" in dependency injection.

Reply:
On second thought, I might have been wrong about `@Inject` not being mandatory for Guice, since looking closer the modified `Dispute*` classes all have subclasses with `@Inject`-annotated constructors - probably only `TransactionsView` and `TakeOfferView` actually needed fixing. (I'm pretty sure it isn't mandatory by default for Spring, but perhaps Guice is stricter.)

Also, there are (probably leftover) `@FxmlView` annotations on the 3 `Dispute*` classes which mislead me to think that they were concrete components - perhaps those annotations can be removed?

Reply:
I think here it is not necessary as mentioned by @lusarz. At least it didn't crash for me here.

Reply:
> But it turns out `@inject` is not infact mandatory and the program crashes on these views without this annotation.

Did it crash for you while testing?

Reply:
I agree that probably only these two views needs to be annotated. 

Yeah, I suppose `@FxmlView` on abstract views are leftovers, and it would be nice to remove them.

Reply:
Inject annotation is needed for the concrete classes but not for the base classes (should not be used there). It probably crashed due some nullpointers caused if the annotation is missing but with view classed you don't get proper error logs.

Reply:
Inject annotation is needed for the concrete classes but not for the base classes (should not be used there). It probably crashed due some nullpointers caused if the annotation is missing but with view classed you don't get proper error logs.